sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java

```diff
@@ -6,61 +6,26 @@
  */
 final class SystematicSensitivityService {
     private static final Logger LOGGER = LoggerFactory.getLogger(SystematicSensitivityService.class);
 
-    private static SensitivityComputationFactory sensitivityComputationFactory;
-    private static ComputationManager computationManager;
-
-    static void init(SensitivityComputationFactory factory, ComputationManager computationManager) {
-        sensitivityComputationFactory = factory;
-        SystematicSensitivityService.computationManager = computationManager;
-    }
-
     static SystematicSensitivityResult runSensitivity(Network network,
                                                                String workingStateId,
                                                                CnecSensitivityProvider cnecSensitivityProvider,
-                                                               SensitivityComputationParameters sensitivityComputationParameters,
-                                                               ComputationManager computationManager) {
-        if (!initialised()) {
-            init(ComponentDefaultConfig.load().newFactoryImpl(SensitivityComputationFactory.class), computationManager);
-        }
-        SensitivityComputation computation = sensitivityComputationFactory.create(network, computationManager, 1);
+                                                               SensitivityAnalysisParameters sensitivityComputationParameters) {
         LOGGER.debug("Sensitivity computation [start]");
-        CompletableFuture<SensitivityComputationResults> results = computation.run(cnecSensitivityProvider, cnecSensitivityProvider, workingStateId, sensitivityComputationParameters);
-        try {
-            SensitivityComputationResults joinedResults = results.join();
-            LOGGER.debug("Sensitivity computation [end]");
-            return new SystematicSensitivityResult(joinedResults);
-        } catch (CompletionException e) {
-            throw new SensitivityComputationException("Sensitivity computation failed");
-        }
-    }
-
-    static SystematicSensitivityResult runSensitivity(Network network,
-                                                               String workingStateId,
-                                                               CnecSensitivityProvider cnecSensitivityProvider,
-                                                               SensitivityComputationParameters sensitivityComputationParameters) {
-        return runSensitivity(network, workingStateId, cnecSensitivityProvider, sensitivityComputationParameters, DefaultComputationManagerConfig.load().createLongTimeExecutionComputationManager());
-    }
-
-    private static boolean initialised() {
-        return sensitivityComputationFactory != null && computationManager != null;
+        SensitivityAnalysisResult result = SensitivityAnalysis.run(network, workingStateId, cnecSensitivityProvider, cnecSensitivityProvider, sensitivityComputationParameters);
```

Comment:
Can we even remove workingStateId ? As we always use network working variant

Reply:
done